packages/core/src/new-api/internal/module-builder.ts

```diff
@@ -357,7 +369,7 @@ export class IgnitionModuleBuilderImplementation<
 
   public contractAt<ContractNameT extends string>(
     contractName: ContractNameT,
-    address: string | NamedStaticCallFuture<string, string>,
+    address: string | AddressResolvableFuture,
```

Comment:
I updated some types that were wrong. It had no impact in the code apart from the types.